go/enclave/db/sql/sqlite.go

```diff
@@ -62,6 +62,15 @@ func CreateTemporarySQLiteDB(dbPath string, logger gethlog.Logger) (*EnclaveDB,
 		dbPath = tempPath
 	}
 	inMem := strings.Contains(dbPath, "mode=memory")
+	description := "in memory"
```

Comment:
Moved this block up because have to check if db file exists before it gets initialised

Reply:
I don't see that info being used . Am I missing something?